src/screens/MessageDetails.js

```diff
@@ -51,10 +51,6 @@ export default class MessageDetails extends React.PureComponent {
                 message={isU8a(message) ? u8aToHex(message) : message}
                 dataToSign={isU8a(dataToSign) ? u8aToHex(dataToSign) : dataToSign}
                 isHash={scannerStore.getIsHash()}
-                onPressAccount={async account => {
```

Comment:
This is an unwanted behaviour that is not related to this PR. Users shouldn't be able to click on an account card in the message details view.